plugins/vendor/sso/services/SsoService.php

```diff
@@ -0,0 +1,149 @@
+<?php
+/**
+ * @service sso
+ * @package plugins.sso
+ * @subpackage api.services
+ */
+class SsoService extends KalturaBaseService
+{
+	/**

```

Comment:
Add init service to apply criteria filter and check is feature allowed (similar to)
plugins/reach/services/ReachProfileService.php